src/Uno.UI/Microsoft/UI/Xaml/Controls/Expander/Expander.cs

```diff
@@ -3,10 +3,11 @@
 using Windows.UI.Xaml.Automation.Peers;
 using Windows.UI.Xaml.Controls;
 using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Markup;
 
 namespace Microsoft.UI.Xaml.Controls
 {
-	public partial class Expander : Control
+	public partial class Expander : ContentControl
```

Comment:
Dumb mistake on my part. Hopefully didn't take too long to track down.

Reply:
Happens 😀 . It was a bit tricky, but @carldebilly helped me notice it 😀 .

Reply:
@MartinZikmund Well, sorry about that!

Reply:
No problem at all 😀 !